pipelines/build/Jenkinsfile

```diff
@@ -4,17 +4,17 @@ properties([
         choiceParam(
             description: 'Release or Dev branch to build',
             name: 'Branch',
-            choices: 'branch-3.0\nbranch-2.8\nmaster'
+            choices: 'branch-3.1\nbranch-3.0\nbranch-2.8\nmaster'
         ),
         choiceParam(
             description: 'Avaliable platforms to build off of',
             name: 'Platform',
-            choices: 'cdh5.12.0\ncdh5.12.2\ncdh5.12.2-2.3\ncdh5.13.3\ncdh5.14.0\ncdh5.14.2\ncdh5.16.1\ncdh6.3.0\nhdp2.6.1\nhdp2.6.3\nhdp2.6.4\nhdp2.6.5\nhdp3.1.0'
+            choices: 'cdh5.12.0\ncdh5.12.2\ncdh5.12.2-2.3\ncdh5.13.3\ncdh5.14.0\ncdh5.14.2\ncdh5.16.1\ncdh6.3.0\nhdp2.6.1\nhdp2.6.3\nhdp2.6.4\nhdp2.6.5\nhdp3.1.0\nhdp3.1.5'
         ),
```

Comment:
Can old cdh version be removed?

Reply:
Okay, I removed the option to select the branch-2.8 platforms. I kept the conditional logic inside the pipeline to switch platforms if we ever need to add it back in support.